src/2025h2/finishing-gpu-offload.md

```diff
@@ -0,0 +1,65 @@
+# Finish the std::offload module
+
+| Metadata              |                                                  |
+| :-------------------- | -------------------------------------------------|
+| Point of contact      | @ZuseZ4                                          |
+| Teams                 | <!-- TEAMS WITH ASKS -->                         |
+| Task owners           | <!-- TASK OWNERS -->                             |
+| Status                | Proposed                                         |
+| Tracking issue        | [rust-lang/rust-project-goals#109]               |
+| Other tracking issues | [rust-lang/rust#124509], [rust-lang/rust#124509] |
```

Comment:
@ZuseZ4 the same issue is listed twice. Is there a separate tracking issue or should we remove the extra?